src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +299,31 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
```

Comment:
We ensure that only when there are multiple merge nodes, this check will still be useful for the following example:
```yml
foo:
    <<: ~
```